src/Utils/Hooks/useAuthIntent/index.tsx

```diff
@@ -65,35 +75,66 @@ export const runAuthIntent = async (
             case "profile":
               return followProfileMutation(relayEnvironment, value.objectId)
           }
+
           break
+
         case "save":
-          triggerEvent(mediator, "artwork:save", value)
           return saveArtworkMutation(relayEnvironment, value.objectId)
+
+        case "associateSubmission":
+          return associateSubmissionMutation(relayEnvironment, value.objectId)
```

Comment:
ok never mind my comment above, I just saw this 👍

Reply:
I was unable to test this end to end because the form is unable to validate sometimes. Going to assume these mutations work.